corehq/apps/custom_data_fields/models.py

```diff
@@ -93,6 +93,7 @@ def __repr__(self):
 class CustomDataFieldsDefinition(models.Model):
     field_type = models.CharField(max_length=126)
     domain = models.CharField(max_length=255, null=True)
+    profile_required_for_user_type = models.CharField(max_length=126, null=True)
```

Comment:
Shouldn't this be a JSONField since it's storing a list of strings? I see in custom_data_fields.js that you have to split the string by `,` to make it into a list.

Another note, if we were to use `CharField`, I think[ recommended practice ](https://docs.djangoproject.com/en/5.1/ref/models/fields/#null)is to not use `null=True` to avoid two possible values of "no data"

Reply:
Thanks for flagging the issue of two possible values of "no data".
Right now it is storing one string. 'web_user,commcare_user', 'web_user', 'commcare_user' or 'neither'. 
Open to converting to a json field. Looking now at what would need to change.

Reply:
Cool, on that similar note, what do you do you think about making the value for "neither" as just an empty list? instead of `["neither"]`

Reply:
yeah, an empty list list makes sense for that